internal/services/loganalytics/log_analytics_workspace_resource.go

```diff
@@ -61,6 +63,12 @@ func resourceLogAnalyticsWorkspace() *pluginsdk.Resource {
 
 			"resource_group_name": azure.SchemaResourceGroupNameDiffSuppress(),
 
+			"allow_resource_only_permissions": {
+				Type:     pluginsdk.TypeBool,
+				Optional: true,
+				Default:  true,
```

Comment:
If the API is defaulting this to false, then we should also be defaulting to false.

Reply:
the service defaults to true. create without this property and we can see it was set to true.

Reply:
what happens when you create it using a previous provider version?

Reply:
tried with the current main branch with API version 2020-08-01, this property defaults to true